content/en/real_user_monitoring/browser/monitoring_vitals_performance.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Monitoring Vitals Performance
+kind: documentation
+further_reading:
+  - link: "https://learn.datadoghq.com/courses/core-web-vitals-lab"
+    tag: "Learning Center"
+    text: "Interactive Lab: Core Web Vitals"
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "https://www.datadoghq.com/blog/core-web-vitals-monitoring-datadog-rum-synthetics/"
+    tag: "Blog"
+    text: "Monitor Core Web Vitals with Datadog RUM and Synthetic Monitoring"
+---
+
+{{< beta-callout url="https://www.google.com" >}}
+Try the beta! Vitals are in public beta.
+{{< /beta-callout >}} 
+
+## Overview
+
+{{< img src="real_user_monitoring/browser/vitals_performance/vitals-results.png" alt="Select a view to start improving your performance vitals." style="width:100%;" >}}
+
+RUM Vitals helps you find the true root cause of browser performance issues based on real user traffic. You can troubleshoot browser metrics, including the [Core Web Vitals][1] (CWV), [Loading Time][2] (Datadog's custom metric that evaluates how long it takes for a page to fully load from the user's perspective), and more. Determine what caused a slow page, with all the relevant information in one place.
+
+## Prerequisites
+
+To get the most out of this feature, Datadog recommends that you use:
+
+- RUM Browser SDK version 5.4.0 or newer
+- Session Replay for at least some sessions
+
+## Select a vital
+
+Navigate to the [Vitals page][3], or click the "Vitals" tab under the [**Digital Experience > Performance Monitoring**][4] page.
+
+{{< img src="real_user_monitoring/browser/vitals_performance/vitals-page-selectors.png" alt="You can check vitals for the top most visited pages or specific pages." style="width:100%;" >}}
+
+From this view, you have two ways of selecting the page and vital that you can review and optimize:
+
+- Select from a list of the most visited pages
+- Enter a view name on the left and select the page
+
+## Filter and evaluate
+
+Once you've selected a page and vital, you can see insights that explain their performance.
+
+From here, you can:
+
```

Comment:
Recommend consistent punctuation format for bullet lists. This one uses punctuation, but the others don't.

Reply:
Ugh yeah I struggled with this one because I only have one bullet point on this page that needs periods.